src/components/graph-view.js

```diff
@@ -1360,14 +1361,20 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     const customContainerId = `${id}-custom-container`;
     const { draggedEdge } = this.state;
     const { afterRenderEdge } = this.props;
-    let edgeContainer = document.getElementById(containerId);
+    let edgeContainer = this.viewWrapper.current.querySelector(
+      `[id='${containerId}']`
+    );
 
     if (nodeMoving && edgeContainer) {
       edgeContainer.style.display = 'none';
       containerId = `${id}-custom-container`;
-      edgeContainer = document.getElementById(containerId);
+      edgeContainer = this.viewWrapper.current.querySelector(
+        `[id='${containerId}']`
+      );
     } else if (edgeContainer) {
-      const customContainer = document.getElementById(customContainerId);
+      const customContainer = this.viewWrapper.current.querySelector(
+        `[id='${customContainerId}']`
+      );
 
       edgeContainer.style.display = '';
 
```

Comment:
Do we need to change 
https://github.com/uber/react-digraph/blob/94bd6525f459615316c06c64da779b66d1c0a136/src/components/graph-view.js#L190-L191
as well?

Reply:
Thanks for checking. I'll fix this.

Reply:
Fixed.